spec/defines/openvpn_server_spec.rb

```diff
@@ -110,6 +110,81 @@
         it { is_expected.to contain_file('/etc/openvpn/test_server.conf').with_content(%r{^rcvbuf\s+393215$}) }
       end
 
+      context 'when using udp4' do
+        let(:params) do
+          {
+            'country'       => 'CO',
+            'province'      => 'ST',
+            'city'          => 'Some City',
+            'organization'  => 'example.org',
+            'email'         => 'testemail@example.org',
+            'proto'         => 'udp4'
+          }
+        end
+
+        it { is_expected.to contain_file('/etc/openvpn/test_server.conf').with_content(%r{^proto\s+udp4$}) }
+      end
+
+      context 'when using udp6' do
+        let(:params) do
+          {
+            'country'       => 'CO',
+            'province'      => 'ST',
+            'city'          => 'Some City',
+            'organization'  => 'example.org',
+            'email'         => 'testemail@example.org',
+            'proto'         => 'udp6'
+          }
+        end
+
+        it { is_expected.to contain_file('/etc/openvpn/test_server.conf').with_content(%r{^proto\s+udp6$}) }
+      end
+
+      context 'when using tcp4' do
+        let(:params) do
+          {
+            'country'       => 'CO',
+            'province'      => 'ST',
+            'city'          => 'Some City',
+            'organization'  => 'example.org',
+            'email'         => 'testemail@example.org',
+            'proto'         => 'tcp4'
+          }
+        end
+
+        it { is_expected.to contain_file('/etc/openvpn/test_server.conf').with_content(%r{^proto\s+tcp4-server$}) }
+      end
+
+      context 'when using tcp6' do
+        let(:params) do
+          {
+              'country'       => 'CO',
```

Comment:
this section has four spaces instead of two and is throwing style errors at https://travis-ci.org/voxpupuli/puppet-openvpn/jobs/488140662

Reply:
Ouuch. Why rubocop is not executed here?